lib/fork_foreman.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const {fork} = require('child_process')
+const path = require('path')
+
+module.exports = function (argv) {
+  let script = path.join(__dirname, '../node_modules/.bin/nf')
```

Comment:
@RasPhilCo can you make this run a script that does a `require('foreman/nf.js)` like before because this is depending on behavior that is not consistent.